addon/ember-data.js

```diff
@@ -21,26 +21,27 @@ let DsSerializers, Serializers
  * Get all ember data models under the app's namespaces
  *
  * @method getDsModels
+ * @param  {Object} options
+ * @param  {array} options.moduleMap - The requirejs.entries map to use
  * @private
  * @hide
  * @return {Object} models
  */
-export function getDsModels() {
+export function getDsModels(options) {
   if (DsModels) {
     return DsModels;
   }
 
-  let moduleMap = requirejs.entries;
   let classicModelMatchRegex = new RegExp(`^${modulePrefix}/models/(.*)$`, 'i');
   let podModelMatchRegex = new RegExp(`^${podModulePrefix || modulePrefix}/(.*)/model$`, 'i');
 
   DsModels = {};
 
-  if (!hasEmberData) {
+  if (!hasEmberData(options)) {
     return DsModels;
   }
 
-  Object.keys(moduleMap)
+  Object.keys(options.moduleMap)
```

Comment:
what happens if `options` is `undefined`? I guess requiring it might be a breaking change since this is an exported function?

Reply:
I guess I didnt notice it was exported. Its only ever called within the same file. I guess it was exported in case someone outside wanted to call it, but I can imagine doing that. Probably shouldnt have been exported